src/sentry/integrations/pipeline.py

```diff
@@ -62,7 +62,8 @@ def _finish_pipeline(self, data):
                 id=data['reinstall_id'],
             )
             integration.update(external_id=data['external_id'], status=ObjectStatus.VISIBLE)
-            integration.get_installation().reinstall()
+            # this can't be scoped to organization
+            integration.get_installation(None).reinstall()
```

Comment:
wasn't totally sure what to do here since i don't think we have the org context?

Reply:
I think the pipeline should have the organization context on the `organization` property.